docs/reference/modules/node.asciidoc

```diff
@@ -89,93 +91,85 @@ memory and CPU in order to deal with the gather phase.
 ===============================================
 
 [[master-node]]
-==== Master-eligible node
+==== Manager node
```

Comment:
This section should really be about control nodes, although the text today is actually rather focussed on the manager responsibilities.